core/src/main/java/nl/nn/adapterframework/lifecycle/ServletManager.java

```diff
@@ -156,14 +156,22 @@ private void registerServlet(Servlet servlet, String servletName, String urlMapp
 		if(!appConstants.getBoolean(propertyPrefix+"enabled", true))
 			return;
 
-		getServletContext().log("instantiating IbisInitializer servlet ["+servletName+"]");
+		ServletSecurityElement security = getServletSecurity(propertyPrefix, roles);
+		StringBuilder builder = new StringBuilder("instantiating IbisInitializer servlet ["+servletName+"] using protocol ");
+		builder.append((security.getTransportGuarantee()==TransportGuarantee.CONFIDENTIAL)?"[HTTPS]":"[HTTP]");
+		if(security.getRolesAllowed().length > 0) {
+			builder.append(" and roles " + Arrays.asList(security.getRolesAllowed()));
+		} else if(!"LOC".equalsIgnoreCase(appConstants.getProperty("dtap.stage"))) {
+			builder.append(" with no authentication enabled!");
```

Comment:
We kunnen er ook voor kiezen bij 'LOC' lokale users te gebruiken, zeg 'admim/admin'